src/main/java/org/primeframework/mvc/cors/CORSConfiguration.java

```diff
@@ -98,8 +105,15 @@ public CORSConfiguration withDebug(boolean debug) {
     return this;
   }
 
+  public CORSConfiguration withExcludeUriPredicate(Predicate<String> excludeUriPredicate) {
```

Comment:
Naming. This should be `withExcludeURIPredicate`, same for other similar names. `Uri` -> `URI`.